examples/run_models/configs/hyena_dna_config.yaml

```diff
@@ -0,0 +1,25 @@
+model_name: "hyenadna-tiny-1k-seqlen-d256"
+n_layer: 2
+vocab_size: 12
+resid_dropout: 0.0
+embed_dropout: 0.1
+fused_mlp: False,   
+fused_dropout_add_ln: True
+residual_in_fp32: True
+checkpoint_mixer: False
+checkpoint_mlp: False
+pad_vocab_size_multiple: 8
+return_hidden_state: True
+device: "cuda"
```

Comment:
should we leave cuda as default?

Reply:
@bputzeys  just wondering if it would crash or not if we don't have it

Reply:
yeah I put it as default now for the checks to run quickly but it doesnt make much of a dfiference because the data is just 10 observations.
I think I will revert to using cpu as default again so that people without GPU can run it straight away